librz/cons/cons.c

```diff
@@ -1457,7 +1458,7 @@ RZ_API int rz_cons_get_size(int *rows) {
 		I.columns = csbi.srWindow.Right - csbi.srWindow.Left + 1;
 		I.rows = csbi.srWindow.Bottom - csbi.srWindow.Top + 1;
 	} else {
-		if (I.term_xterm) {
+		if (I.term_pty) {
 			ret = __xterm_get_size();
```

Comment:
If you rename `term_xterm` to `term_pty`, maybe it makes sense also to rename this function?